tslint.json

```diff
@@ -66,12 +66,7 @@
             "check-open-brace",
             "check-whitespace"
         ],
-        "ordered-imports": [
-            true,
-            {
-                "named-imports-order": "lowercase-last"
-            }
-        ],
+        "ordered-imports": false,
```

Comment:
Let's not change this please. I want to keep the imports in order. I was talking about @NejcZdovc that current order is maybe not the best, but it is still better for me than random. If you want us to change it, open an issue and we can discuss the options there.